starter-code/public/scripts/articleView.js

```diff
@@ -131,7 +130,7 @@ articleView.initAdminPage = function() {
   // TODO: Call the Handlebars `.compile` function, which will return a function for you to use where needed.
   // Make sure you assign the result of your Handlebars.compile call to a variable called "template", since
   // we are then calling "template" on line 117.
-
+var template = Handlebars.compile($('#admin-template').text());
```

Comment:
So I don't think you are selecting the correct template? In your index.html file the template is called author-template, but here you are selecting admin-template.
Point deduction: -1